### PR TITLE
python3Packages.autobahn: 20.12.3 -> 21.3.1

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 21.3.1.

Change log: https://autobahn.readthedocs.io/en/latest/changelog.html#id2

Also, ran `nixpkgs-fmt`.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112318` at f5e15c9d run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>29 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/761835f6d1c66ae9320c716d0b7321a3">buildbot</a></li> (python38Packages.buildbot)</li>
    <li><a href="https://gist.github.com/2f8e6991e4a8972584c0c18f8d22653c">buildbot-full</a></li> (python38Packages.buildbot-full)</li>
    <li><a href="https://gist.github.com/7f7fa4f396c1e2e813575a70b40dcd47">buildbot-ui</a></li> (python38Packages.buildbot-ui)</li>
    <li><a href="https://gist.github.com/560a1d219eac4f0d1084eec7779b5ac2">gnome-keysign</a></li></li>
    <li><a href="https://gist.github.com/ca3c57e11934c985ad74677cf265e473">grab-site</a></li></li>
    <li><a href="https://gist.github.com/faa6b4128ac06d478b0ec66153a6c50a">magic-wormhole</a></li> (python38Packages.magic-wormhole)</li>
    <li><a href="https://gist.github.com/b3bbf4c7635ac33303d3c919b3066b17">python37Packages.autobahn</a></li></li>
    <li><a href="https://gist.github.com/cc3d99ea185546331ef079f8e87e1f22">python37Packages.buildbot</a></li></li>
    <li><a href="https://gist.github.com/33b232e4ded41404e9f225107aea0f22">python37Packages.buildbot-full</a></li></li>
    <li><a href="https://gist.github.com/ed1605b8b1681f1c24ebdf4f2331dad4">python37Packages.buildbot-ui</a></li></li>
    <li><a href="https://gist.github.com/1ee2fc40595476de128cf27892412658">python37Packages.channels</a></li></li>
    <li><a href="https://gist.github.com/b7cb5bcd87202969ba0547c897e61861">python37Packages.channels-redis</a></li></li>
    <li><a href="https://gist.github.com/7c5fd444484b2379b00b0ea94675fea7">python37Packages.daphne</a></li></li>
    <li><a href="https://gist.github.com/a58bf9463aa0e1927431e784ef674040">python37Packages.magic-wormhole</a></li></li>
    <li><a href="https://gist.github.com/320cf7b25051e4b6c4985bf391ba5b5f">python37Packages.magic-wormhole-mailbox-server</a></li></li>
    <li><a href="https://gist.github.com/7d9e660ddf84a65863fe2069f848695e">python37Packages.python-binance</a></li></li>
    <li><a href="https://gist.github.com/174a0c40647cf13824709de9c24c8cfc">python38Packages.autobahn</a></li></li>
    <li><a href="https://gist.github.com/9272678ef06d55c8702f41dd26dd8afa">python38Packages.channels</a></li></li>
    <li><a href="https://gist.github.com/222df01b9c372bf170bc323b611d6a38">python38Packages.channels-redis</a></li></li>
    <li><a href="https://gist.github.com/3264c8f692a5e35ff952165227fc4b03">python38Packages.daphne</a></li></li>
    <li><a href="https://gist.github.com/4268c0be2dde64fc1d6365b8d72d00be">python38Packages.magic-wormhole-mailbox-server</a></li></li>
    <li><a href="https://gist.github.com/5ff6b8fdd766fd1cb97c506c216bb955">python38Packages.python-binance</a></li></li>
    <li><a href="https://gist.github.com/c6bbc97b1c7b135d5f799600c8e2a4b5">python39Packages.autobahn</a></li></li>
    <li><a href="https://gist.github.com/167b8231d92b6a754066c59eeae757cc">python39Packages.channels</a></li></li>
    <li><a href="https://gist.github.com/6f911ce39b825fce50c999bcceea8dd5">python39Packages.channels-redis</a></li></li>
    <li><a href="https://gist.github.com/a4cee71b4972aeb82473ab4995bbb60b">python39Packages.daphne</a></li></li>
    <li><a href="https://gist.github.com/4ca2a790db6c09bdc91dc45586cb9287">python39Packages.magic-wormhole</a></li></li>
    <li><a href="https://gist.github.com/f0151b57d3d4ed2d84cd83664cc48db1">python39Packages.magic-wormhole-mailbox-server</a></li></li>
    <li><a href="https://gist.github.com/8e1459c709bb6f6d96411df9e8c999c2">python39Packages.python-binance</a></li></li>
  </ul>
</details>
<details>
  <summary>3 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-explicit-check-phase.md">python-explicit-check-phase</a>

Consider using `pytestCheckHook` in `checkInputs` rather than `checkPhase = "pytest";`.

Near pkgs/development/python-modules/magic-wormhole-mailbox-server/default.nix:17:3:
```
   |
17 |   checkPhase = ''
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md">python-include-tests</a>

Add a `checkPhase` for tests, or at least `pythonImportsCheck`.
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl2` is a deprecated license, check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/development/python-modules/buildbot/default.nix:101:7:
```
    |
101 |       license = licenses.gpl2;
    |       ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/autobahn/default.nix [hunk 1]
```diff
@@ -24,11 +37,12 @@ buildPythonPackage rec {
   # Tests do no seem to be compatible yet with pytest 5.1
   # https://github.com/crossbario/autobahn-python/issues/1235
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/autobahn/default.nix:38]: This is is closed. Do the tests work now?

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 112318` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>32 packages built:</summary>
  <ul>
    <li>buildbot (python38Packages.buildbot)</li>
    <li>buildbot-full (python38Packages.buildbot-full)</li>
    <li>buildbot-ui (python38Packages.buildbot-ui)</li>
    <li>gnome-keysign</li>
    <li>grab-site</li>
    <li>magic-wormhole (python38Packages.magic-wormhole)</li>
    <li>python37Packages.autobahn</li>
    <li>python37Packages.buildbot</li>
    <li>python37Packages.buildbot-full</li>
    <li>python37Packages.buildbot-ui</li>
    <li>python37Packages.channels</li>
    <li>python37Packages.channels-redis</li>
    <li>python37Packages.daphne</li>
    <li>python37Packages.labgrid</li>
    <li>python37Packages.magic-wormhole</li>
    <li>python37Packages.magic-wormhole-mailbox-server</li>
    <li>python37Packages.python-binance</li>
    <li>python38Packages.autobahn</li>
    <li>python38Packages.channels</li>
    <li>python38Packages.channels-redis</li>
    <li>python38Packages.daphne</li>
    <li>python38Packages.labgrid</li>
    <li>python38Packages.magic-wormhole-mailbox-server</li>
    <li>python38Packages.python-binance</li>
    <li>python39Packages.autobahn</li>
    <li>python39Packages.channels</li>
    <li>python39Packages.channels-redis</li>
    <li>python39Packages.daphne</li>
    <li>python39Packages.labgrid</li>
    <li>python39Packages.magic-wormhole</li>
    <li>python39Packages.magic-wormhole-mailbox-server</li>
    <li>python39Packages.python-binance</li>
  </ul>
</details>

File: pkgs/development/python-modules/autobahn/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/development/python-modules/autobahn/default.nix]: Partially at least they do.

File: pkgs/development/python-modules/autobahn/default.nix [hunk 2]
```diff
@@ -1,34 +1,67 @@
...
+    six
+    txaio
+    twisted
+    zope_interface
+    cffi
+    cryptography
+    pynacl
+  ] ++ (lib.optionals (!isPy3k) [
+    trollius
+    futures
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/autobahn/default.nix:38]:
```suggestion
```

upstream does no longer list python2 as supported.

File: pkgs/development/python-modules/autobahn/default.nix [hunk 3]
```diff
@@ -1,34 +1,67 @@
-{ lib, buildPythonPackage, fetchPypi, isPy3k,
-  six, txaio, twisted, zope_interface, cffi, trollius, futures,
-  mock, pytest, cryptography, pynacl
+{ lib
+, buildPythonPackage
+, cffi
+, cryptography
+, fetchPypi
+, futures
+, isPy3k
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/autobahn/default.nix:7]:
```suggestion
```

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 112318` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package marked as broken and skipped:
</summary>
<ul>
<li>python39Packages.magic-wormhole</li>
</ul>
</details>
<details>
<summary>
31 packages built:
</summary>
<ul>
<li>buildbot (python38Packages.buildbot)</li>
<li>buildbot-full (python38Packages.buildbot-full)</li>
<li>buildbot-ui (python38Packages.buildbot-ui)</li>
<li>gnome-keysign</li>
<li>grab-site</li>
<li>magic-wormhole (python38Packages.magic-wormhole)</li>
<li>python37Packages.autobahn</li>
<li>python37Packages.buildbot</li>
<li>python37Packages.buildbot-full</li>
<li>python37Packages.buildbot-ui</li>
<li>python37Packages.channels</li>
<li>python37Packages.channels-redis</li>
<li>python37Packages.daphne</li>
<li>python37Packages.labgrid</li>
<li>python37Packages.magic-wormhole</li>
<li>python37Packages.magic-wormhole-mailbox-server</li>
<li>python37Packages.python-binance</li>
<li>python38Packages.autobahn</li>
<li>python38Packages.channels</li>
<li>python38Packages.channels-redis</li>
<li>python38Packages.daphne</li>
<li>python38Packages.labgrid</li>
<li>python38Packages.magic-wormhole-mailbox-server</li>
<li>python38Packages.python-binance</li>
<li>python39Packages.autobahn</li>
<li>python39Packages.channels</li>
<li>python39Packages.channels-redis</li>
<li>python39Packages.daphne</li>
<li>python39Packages.labgrid</li>
<li>python39Packages.magic-wormhole-mailbox-server</li>
<li>python39Packages.python-binance</li>
</ul>
</details>

File: pkgs/development/python-modules/autobahn/default.nix (hunk 2 above)
Comment (fabaff) [reply pkgs/development/python-modules/autobahn/default.nix]: Indeed, thanks.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 112318` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>22 packages built:</summary>
  <ul>
    <li>buildbot (python38Packages.buildbot)</li>
    <li>buildbot-full (python38Packages.buildbot-full)</li>
    <li>buildbot-ui (python38Packages.buildbot-ui)</li>
    <li>gnome-keysign</li>
    <li>grab-site</li>
    <li>magic-wormhole (python38Packages.magic-wormhole)</li>
    <li>python38Packages.autobahn</li>
    <li>python38Packages.channels</li>
    <li>python38Packages.channels-redis</li>
    <li>python38Packages.daphne</li>
    <li>python38Packages.labgrid</li>
    <li>python38Packages.magic-wormhole-mailbox-server</li>
    <li>python38Packages.python-binance</li>
    <li>python38Packages.txaio</li>
    <li>python39Packages.autobahn</li>
    <li>python39Packages.channels</li>
    <li>python39Packages.channels-redis</li>
    <li>python39Packages.daphne</li>
    <li>python39Packages.labgrid</li>
    <li>python39Packages.magic-wormhole-mailbox-server</li>
    <li>python39Packages.python-binance</li>
    <li>python39Packages.txaio</li>
  </ul>
</details>

File: pkgs/development/python-modules/autobahn/default.nix [hunk 4]
```diff
@@ -1,34 +1,80 @@
...
+, futures
+, mock
+, msgpack
+, passlib
+, pynacl
+, pytest-asyncio
+, pytestCheckHook
+, pythonOlder
+, six
+, trollius
```
Comment (wamserma) [inline pkgs/development/python-modules/autobahn/default.nix:20]: six and trollius are no longer required

File: pkgs/development/python-modules/autobahn/default.nix [hunk 5]
```diff
@@ -1,34 +1,80 @@
...
+, argon2_cffi
+, attrs
+, buildPythonPackage
+, cbor
+, cbor2
+, cffi
+, cryptography
+, fetchPypi
+, flatbuffers
+, futures
```
Comment (wamserma) [inline pkgs/development/python-modules/autobahn/default.nix:11]: futures is no longer required

File: pkgs/development/python-modules/autobahn/default.nix [hunk 6]
```diff
@@ -1,34 +1,80 @@
...
+    argon2_cffi
+    cbor
+    cbor2
+    cffi
+    cryptography
+    flatbuffers
+    msgpack
+    passlib
+    pynacl
+    six
```
Comment (wamserma) [inline pkgs/development/python-modules/autobahn/default.nix:47]: six can be removed

File: pkgs/development/python-modules/autobahn/default.nix [hunk 7]
```diff
@@ -1,34 +1,80 @@
...
+, twisted
+, txaio
+, ujson
+, zope_interface
 }:
+
 buildPythonPackage rec {
   pname = "autobahn";
-  version = "20.12.3";
+  version = "21.2.1";
```
Comment (wamserma) [inline pkgs/development/python-modules/autobahn/default.nix:29]: latest is 21.3.1, maybe update the PR and feel free to cherry-pick https://github.com/NixOS/nixpkgs/pull/116971/commits/e99732a96cf6715277db7d1db89763b37150611c into this PR

**Review by wamserma — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 112318` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>2 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>python38Packages.labgrid: log was empty</li>
<li>python39Packages.labgrid: log was empty</li>
  </ul>
</details>
<details>
<summary>
20 packages built:
</summary>
<ul>
<li>buildbot (python38Packages.buildbot)</li>
<li>buildbot-full (python38Packages.buildbot-full)</li>
<li>buildbot-ui (python38Packages.buildbot-ui)</li>
<li>gnome-keysign</li>
<li>grab-site</li>
<li>magic-wormhole (python38Packages.magic-wormhole)</li>
<li>python38Packages.autobahn</li>
<li>python38Packages.channels</li>
<li>python38Packages.channels-redis</li>
<li>python38Packages.daphne</li>
<li>python38Packages.magic-wormhole-mailbox-server</li>
<li>python38Packages.python-binance</li>
<li>python38Packages.txaio</li>
<li>python39Packages.autobahn</li>
<li>python39Packages.channels</li>
<li>python39Packages.channels-redis</li>
<li>python39Packages.daphne</li>
<li>python39Packages.magic-wormhole-mailbox-server</li>
<li>python39Packages.python-binance</li>
<li>python39Packages.txaio</li>
</ul>
</details>

---

python3.8-pytest-dependency fails to build with many test failures.

File: pkgs/development/python-modules/autobahn/default.nix (hunk 4 above)
Comment (fabaff) [reply pkgs/development/python-modules/autobahn/default.nix:20]: Remove, thanks.

File: pkgs/development/python-modules/autobahn/default.nix (hunk 6 above)
Comment (fabaff) [reply pkgs/development/python-modules/autobahn/default.nix:47]: Remove, thanks.

File: pkgs/development/python-modules/autobahn/default.nix (hunk 5 above)
Comment (fabaff) [reply pkgs/development/python-modules/autobahn/default.nix:11]: Remove, thanks.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 112318` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>python3Packages.autobahn</li>
  </ul>
</details>

File: pkgs/development/python-modules/autobahn/default.nix (hunk 7 above)
Comment (fabaff) [reply pkgs/development/python-modules/autobahn/default.nix]: It's now for 21.3.1 but as a later release of cryptography is required it has to go to `staging`.

File: pkgs/development/python-modules/autobahn/default.nix [hunk 8]
```diff
@@ -1,34 +1,81 @@
...
 
-  # Tests do no seem to be compatible yet with pytest 5.1
-  # https://github.com/crossbario/autobahn-python/issues/1235
-  doCheck = false;
+  pytestFlagsArray = [ "--pyargs autobahn" ];
+
+  disabledTests = [
+    "TestSerializer"
+    "test_basic"
+  ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/autobahn/default.nix:71]: Why are those tests disabled?

Comment (fabaff) [reply pkgs/development/python-modules/autobahn/default.nix:71]: Too bad that I didn't add a comment as now I can't remember.

**Comment by SuperSandro2000** [comment]
@fabaff please fixthe merge conflict.,

File: pkgs/development/python-modules/autobahn/default.nix (hunk 8 above)
Comment (fabaff) [reply pkgs/development/python-modules/autobahn/default.nix]: With an additional module we are able to run all tests. `py-ubjson` was added.

File: pkgs/development/python-modules/hyperlink/default.nix [hunk 9]
```diff
@@ -1,19 +1,26 @@
...
+, buildPythonPackage
+, fetchPypi
+, isPy27
+, idna
+, typing ? null
+}:
 
 buildPythonPackage rec {
   pname = "hyperlink";
   version = "21.0.0";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/hyperlink/default.nix:11]: I think this already got updated. Can you please change the commit to ``cleanups, add homepage``?

Comment (fabaff) [reply pkgs/development/python-modules/hyperlink/default.nix:11]: Done, thanks

**Review by SuperSandro2000 — COMMENTED** [review]
the autobahn commit also requires an update as we already have that version in staging.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 112318` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>1 package failed to build and are new build failure:</summary>
  <ul>
<li>grab-site: log was empty</li>
  </ul>
</details>
<details>
<summary>
27 packages built:
</summary>
<ul>
<li>buildbot (python38Packages.buildbot)</li>
<li>buildbot-full (python38Packages.buildbot-full)</li>
<li>buildbot-ui (python38Packages.buildbot-ui)</li>
<li>gnome-keysign</li>
<li>magic-wormhole (python38Packages.magic-wormhole)</li>
<li>python38Packages.autobahn</li>
<li>python38Packages.channels</li>
<li>python38Packages.channels-redis</li>
<li>python38Packages.daphne</li>
<li>python38Packages.labgrid</li>
<li>python38Packages.magic-wormhole-mailbox-server</li>
<li>python38Packages.py-ubjson</li>
<li>python38Packages.python-binance</li>
<li>python38Packages.txaio</li>
<li>python39Packages.autobahn</li>
<li>python39Packages.buildbot</li>
<li>python39Packages.buildbot-full</li>
<li>python39Packages.buildbot-ui</li>
<li>python39Packages.channels</li>
<li>python39Packages.channels-redis</li>
<li>python39Packages.daphne</li>
<li>python39Packages.labgrid</li>
<li>python39Packages.magic-wormhole</li>
<li>python39Packages.magic-wormhole-mailbox-server</li>
<li>python39Packages.py-ubjson</li>
<li>python39Packages.python-binance</li>
<li>python39Packages.txaio</li>
</ul>
</details>

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 112318` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages failed to build:</summary>
  <ul>
    <li>buildbot (python38Packages.buildbot)</li>
    <li>buildbot-full (python38Packages.buildbot-full)</li>
    <li>buildbot-ui (python38Packages.buildbot-ui)</li>
  </ul>
</details>
<details>
  <summary>25 packages built:</summary>
  <ul>
    <li>gnome-keysign</li>
    <li>grab-site</li>
    <li>magic-wormhole (python38Packages.magic-wormhole)</li>
    <li>python38Packages.autobahn</li>
    <li>python38Packages.channels</li>
    <li>python38Packages.channels-redis</li>
    <li>python38Packages.daphne</li>
    <li>python38Packages.labgrid</li>
    <li>python38Packages.magic-wormhole-mailbox-server</li>
    <li>python38Packages.py-ubjson</li>
    <li>python38Packages.python-binance</li>
    <li>python38Packages.txaio</li>
    <li>python39Packages.autobahn</li>
    <li>python39Packages.buildbot</li>
    <li>python39Packages.buildbot-full</li>
    <li>python39Packages.buildbot-ui</li>
    <li>python39Packages.channels</li>
    <li>python39Packages.channels-redis</li>
    <li>python39Packages.daphne</li>
    <li>python39Packages.labgrid</li>
    <li>python39Packages.magic-wormhole</li>
    <li>python39Packages.magic-wormhole-mailbox-server</li>
    <li>python39Packages.py-ubjson</li>
    <li>python39Packages.python-binance</li>
    <li>python39Packages.txaio</li>
  </ul>
</details>

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 112318` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>28 packages built:</summary>
  <ul>
    <li>buildbot (python38Packages.buildbot)</li>
    <li>buildbot-full (python38Packages.buildbot-full)</li>
    <li>buildbot-ui (python38Packages.buildbot-ui)</li>
    <li>gnome-keysign</li>
    <li>grab-site</li>
    <li>magic-wormhole (python38Packages.magic-wormhole)</li>
    <li>python38Packages.autobahn</li>
    <li>python38Packages.channels</li>
    <li>python38Packages.channels-redis</li>
    <li>python38Packages.daphne</li>
    <li>python38Packages.labgrid</li>
    <li>python38Packages.magic-wormhole-mailbox-server</li>
    <li>python38Packages.py-ubjson</li>
    <li>python38Packages.python-binance</li>
    <li>python38Packages.txaio</li>
    <li>python39Packages.autobahn</li>
    <li>python39Packages.buildbot</li>
    <li>python39Packages.buildbot-full</li>
    <li>python39Packages.buildbot-ui</li>
    <li>python39Packages.channels</li>
    <li>python39Packages.channels-redis</li>
    <li>python39Packages.daphne</li>
    <li>python39Packages.labgrid</li>
    <li>python39Packages.magic-wormhole</li>
    <li>python39Packages.magic-wormhole-mailbox-server</li>
    <li>python39Packages.py-ubjson</li>
    <li>python39Packages.python-binance</li>
    <li>python39Packages.txaio</li>
  </ul>
</details>

**Pull request merged**